examples/NucleoF767/c-leds/readme.md

```diff
@@ -65,4 +65,4 @@ If you have your device connected to the serial port you will see the output of
 
 ## Troubleshooting
 
-If you don't find a conclusion using the [documentation](http://http://kiso.rempler.de:1313), please message us on [mattermost](mattermost.eclipse.org/eclipse/channels/kiso).
\ No newline at end of file
+If you don't find a conclusion using the [documentation](http://http://kiso.rempler.de:1313), please message us on [mattermost](mattermost.eclipse.org/eclipse/channels/kiso).
```

Comment:
second was not fixed--

Reply:
Thanks for noticing. Unfortunately, I messed up my previous fork. Thus, this pull request is linked to an "unknown repository" -> closed and new pull request created